app/services/user_menu.rb

```diff
@@ -0,0 +1,22 @@
+class UserMenu
+  def generate
+    menus = [
+      Menu::Manager.to_hash(:top_menu),
+      Menu::Manager.to_hash(:side_menu),
+      Menu::Manager.to_hash(:admin_menu),
+    ]
+    menus.each_with_object([]) { |menu, memo| memo.concat submenu_items(menu) }
```

Comment:
This looks quite similar to what we have here: https://github.com/theforeman/foreman/blob/develop/app/helpers/layout_helper.rb#L6-L11 
Would it make sense to use the same logic for both places or combine them to one place?